aries-site/src/data/wcag/components.json

```diff
@@ -47,6 +47,412 @@
+        "label": "Time-based Media",
+        "status": "not-applicable"
+      },
+      {
+        "rule": "1.3.1",
+        "label": "Info and Relationships",
+        "status": "passed",
+        "notes": "Uses proper aria roles (`role='tab'`, `role='tablist'`, `role='tabpanel'`), uses aria-label to connect tabs to panels via tab title prop. And uses `aria-selected` to indicate the selected tab."
+      },
+      {
+        "rule": "1.3.2",
+        "label": "Meaningful Sequence",
+        "status": "conditional",
+        "notes": "The implementer needs to right justified the text when in RTL mode"
+      },
+      {
+        "rule": "1.3.3",
+        "label": "Sensory Characteristics",
+        "status": "not-applicable"
+      },
+      {
+        "rule": "1.3.4",
+        "label": "Orientation",
+        "status": "not-applicable"
+      },
+      {
+        "rule": "1.3.5",
+        "label": "Identify Input Purpose",
+        "status": "not-applicable"
+      },
+      {
+        "rule": "1.3.6",
+        "label": "Identify Purpose",
+        "status": "conditional",
+        "notes": "`aria-label` should be provided when tabs are icon only."
+      },
+      {
+        "rule": "1.4.1",
+        "label": "Use of Color",
+        "status": "passed",
+        "notes": "Uses background color to indicate state."
+      },
+      {
+        "rule": "1.4.2",
+        "label": "Audio Control",
+        "status": "not-applicable"
+      },
+      {
+        "rule": "1.4.3",
+        "label": "Contrast (Minimum)",
+        "status": "passed",
+        "notes": "Minimum contrast ratio of 4.61 (selected state) exceeds 4.5:1."
+      },
+      {
+        "rule": "1.4.4",
+        "label": "Resize Text",
+        "status": "passed"
+      },
+      {
+        "rule": "1.4.5",
+        "label": "Images of Text",
+        "status": "not-applicable"
+      },
+      {
+        "rule": "1.4.6",
+        "label": "Contrast (Enhanced)",
+        "status": "failed",
+        "notes": "Minimum contrast ratio of 4.61 (selected state) does not exceed 7:1. But this will pass for larger text which requires 4.5:1."
+      },
+      {
+        "rule": "1.4.7",
+        "label": "Low or No Background Audio",
+        "status": "not-applicable"
+      },
+      {
+        "rule": "1.4.8",
+        "label": "Visual Presentation",
+        "status": "not-applicable"
+      },
+      {
+        "rule": "1.4.9",
+        "label": "Images of Text (No Exception)",
+        "status": "not-applicable"
+      },
+      {
+        "rule": "1.4.10",
+        "label": "Reflow",
+        "status": "passed"
+      },
+      {
+        "rule": "1.4.11",
+        "label": "Non-text Contrast",
+        "status": "passed",
+        "notes": "Minimum contrast ratio of 4.61 exceeds 3:1."
+      },
+      {
+        "rule": "1.4.12",
+        "label": "Text Spacing",
+        "status": "conditional",
+        "notes": "The implementer needs apply required text spacing"
```

Comment:
I don't think this is conditional
Test this with the HPE theme styling and then change the styling of the following:
- Line height (line spacing) to at least 1.5 times the font size;
- Spacing following paragraphs to at least 2 times the font size;
- Letter spacing (tracking) to at least 0.12 times the font size;
- Word spacing to at least 0.16 times the font size.